### ISSUE TITLE
Use `directUrl` in `prisma studio`

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#17358]
4.9.0 added `directUrl` for Migration and Introspection commands. We should also use `directUrl` for local Studio.

**Comment by dkantereivin** [comment]
Seems to have really weird behaviors for me with respect to environment variables. I can try to get an MRE in the next bit.

But generally I'm using data proxy for a current project, I add `directUrl` in the datasource and have it point to an environment variable.

e.g.
```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
  directUrl = env("DIRECT_DATABASE_URL")
}

It works perfectly fine when running migration commands, but when the schema is loaded to actually run queries, it throws on the first request (currently using serverless so `db.$connect` is not invoked manually, but presumably it would throw at this point).

Error is that it couldn't find the `DIRECT_DATABASE_URL` despite the fact that it is set in `.env` and works fine for migration commands.

**Comment by andyjy** [comment]
@dkantereivin was this your issue here? just fixed in 4.10.1 patch release: https://github.com/prisma/prisma/issues/17797

separately, I've subscribed to the original issue here (for prisma studio) which will allow me to simplify my current use of npm package scripts to start the studio after swapping the env var.

**Comment by dkantereivin** [comment]
It doesn't look the same to me: in my case, the error was occuring in dev and was occuring even when all the correct environment variables were set. Prisma just suddenly wasn't seeing some of them.

On the other hand, the linked issue seems to describe the behaviour that prisma requires the environment variables to generate, meaning that a live database must be provided even for mocked tests and similar.

**Comment by janpio** [comment]
I assume you were using the Data Proxy during local development? I think what you describe is indeed what https://github.com/prisma/prisma/issues/17810 referred to @dkantereivin and was also fixed with 4.10.1. An upgrade should help then. (#17797 was more general and not limited to `directUrl` or Data Proxy)

**Comment by janpio** [comment]
First attempt: https://github.com/prisma/prisma/pull/19583 + https://github.com/prisma/studio-code/pull/483
